package/osx/make-package

```diff
@@ -81,6 +82,15 @@ find-program ANT ant            \
    "${HOME}/homebrew/arm64/bin" \
    "${HOME}/opt/bin"
 
+# find node
+find-program NODE node \
+   "../../dependencies/common/node/${RSTUDIO_NODE_VERSION}/bin" \
+   "/opt/rstudio-tools/dependencies/common/node/${RSTUDIO_NODE_VERSION}/bin"
```

Comment:
Does this mean we need to have node.js installed even if we want to build server or QT-based desktop?

Reply:
Yes, but this was already true. The visual editor stuff requires node.js at build time. Only difference is I need to invoke it from cmake; whereas visual editor invokes it via ant (build.xml).